irm/setup.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+
+import setuptools
+
+name = 'google-cloud-irm'
+description = 'Stackdriver Incident Response & Managment API client library'
+version = '0.1.0'
+release_status = 'Development Status :: 3 - Alpha'
+dependencies = [
+    'google-api-core[grpc] >= 1.6.0, < 2.0.0dev',
+    'google-cloud-monitoring',
```

Comment:
IRM has a dependency on monitoring. How should the version be pinned?

Reply:
Ugh, `google-cloud-monitoring` is still in alpha status.  I would say use the current minor release as a minimum, and cap it at the next minor release, e.g.:

```python
    'google-cloud-monitoring >= 0.31.1, < 0.32dev'
```

Reply:
Also, note that you'll need to blacken first, which will morph all those lovely single quotes into doubles. :)